### DISCUSSION TITLE
Please add solid website URL to GitHub repo details

### DISCUSSION THREAD

**Discussion by bobaekang** [solidjs/solid#511] [General]
First of all, huge congrats on 1.0 release :tada:! I've been loosely following this project now for over a year and it's really great to see that the project has hit a major milestone. @ryansolid and key contributors deserve much praise for their accomplishment.

That said, I'd like to ask the maintainers to add solid website url to the repo details, i.e. the "About" section on top right:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/22449454/123700092-a1826900-d825-11eb-8606-d5f328fb0690.png">

This location is often easier to find than somewhere down in the README, and will help direct interested people to more comprehensive materials on Solid when they visit this repo first.

Cheers!

**Comment by ryansolid** [comment]
Thank you. We didn't have a website so long missed all the different places we could link it. And that's a key one. It has been updated.
